wifiphisher/deauth.py

```diff
@@ -7,8 +7,8 @@
 
 import threading
 import scapy.layers.dot11 as dot11
-from scapy.all import *
-from constants import *
+import scapy.arch.linux as linux
+import wifiphisher.constants as constants
```

Comment:
Okay, I will merge this for now.  

What is more important is to stick to the same style all over the code. I would expect a PR that converts all "from constants import *" to "import constants". That's why it's important to only stick to a specific change when you creating PRs.